modules/deploying-resource.adoc

```diff
@@ -12,13 +12,11 @@
 [id="{FeatureName}-deploying_{context}"]
 = Deploying the {FeatureName}
 
-To deploy the {FeatureName}, you create an instance of the `{FeatureName}`
-resource.
+To deploy the {FeatureName}, you create an instance of the `{FeatureName}` resource.
```

Comment:
I'm not sure if I should mess with this in this PR or not. Since `{FeatureName}` is e.g. `ClusterAutoscaler`, this means that the first instance of the `{FeatureName}` will be incorrectly formatted.

Reply:
@bergerhoffer thoughts?

Reply:
I agree that you don't need to take care of it in this PR. I'll follow up on this after this PR is merged. Thanks!

Reply:
@jeana-redhat here's the PR to follow up on this: https://github.com/openshift/openshift-docs/pull/28386